modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -202,7 +202,7 @@ Otherwise it is matched as a case-sensitive string.
 
 | `features`
 | `array (string)`
-| List of additional features to enable. They are all disabled by default. Enabling additional features might have performance impacts. A possible value is `+`.
+| List of additional features to enable. They are all disabled by default. Enabling additional features might have performance impacts. Possible values are: +
 
 - `PacketDrop`: enable the packets drop flows logging feature. This feature requires mounting
```

Comment:
I know many of these sentences are existing text. But, they should start with capital letters, as they are sentences.